.changeset/silly-bees-grin.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
This file is not needed for non package changes such documentation.

Reply:
Tks 😉 I'II delete it